index.md

```diff
@@ -1,11 +1,244 @@
+
+A number of these interventions would have been ineffective for a DR1-scale event.
+
+The eventual goal should be a 3-month worst-case release consisting of 2 months of release work _including_ contingency, and 1 month of testing/bugfixing prior to release. We are unlikely to reach this goal in the next two releases.
+
+## High-level issues, and related recommendations
+
+
+### Handover checkpoints
+
+While it's not quite as simple, there are six major transfers that can serve as checkpoints after data production is complete:
+
+1. Pipelines to Qserv (Catalogs)
+2. Pipelines to Middleware (Images)
+3. Pipelines to [Data Services? Middleware?] (HiPS)
+4. Qserv to/from Data Engineering ()
+5. Data Engineering to SQuaRE/Portal
+6. Data Services to CST
+
+In the data previews so far this has been done "bucket-brigade" style with teams leads at each transfer point co-ordinating the bucket-passing between themselves, as opposed to the Czar model of a single person managing all the steps.
+
+The bucket-brigade model is defensible, as the people in the observatory of the caliber and technical expertise to serve as a Czar are already over-subscribed, and many of these transfers require deep domain knowledge of the two systems involved.
+The cracks in DP1 were less due to the bucket-brigade model itself and more to the ad-hoc definition of who was passing the bucket when, to whom, what was in the bucket anyway, and whose role it was to check that.
+This was compounded by the Data Services Lead failing to understand until late in the game that she needed to be exerting oversight of the entire data release "pipeline", rather that "just" leading RSP service development and data publication.
+
+**Recommendation:**  A "shipping list" is drawn for each of the above six transfer points. The team passing the bucket has the responsibility to check that all the items in the shipment can be checked off. These shipping lists should be templated in some way (Github, JIRA) to be re-used for subsequent releases.
+
+**Recommendation:** Future data release schedules include explicit deadlines for each of those steps, with steps 1-3 being parallelized to the extent possible. Missing these early deadlines should trigger a corresponding schedule delay.
+
+**Recommendation:** A person who is _not_ the Data Services Lead, ideally from Pipelines or V&V should exercise oversight on steps 1-3 above and hand over to the Data Services Lead for steps 4-6.
+
+### Gated approval from a person/body with sign-off powers
+
+While the peer-to-peer bucket-brigade model is well suited for technical handover, there were a number of times where the Data Services Lead, specifically, had to make decisions that were not properly in her purview.
+An example of this was which HiPS product to use for the coverage map, which field to center the coverage map to, etc.
```

Comment:
The PO did a larger fraction of this for DP0.2.  :)

My suggestion here is that there be a discussion _in advance_ (specifically, before the completion of pipeline processing) with stakeholders (e.g., including a representative of the Pipelines team) to discuss what the full presentation of the data _in the data services_ will/should look like.  This is when issues like "we need a HiPS background map for catalog query results in the Portal"; "what HiPS images are attractive and useful representations of the data" could be discussed.

Some of the last-minute decisions referred to here would not have been necessary if this conversation had happened.